tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/B2cAuthorityTests.cs

```diff
@@ -17,42 +15,34 @@ namespace Microsoft.Identity.Test.Unit.CoreTests.InstanceTests
     public class B2CAuthorityTests : TestBase
     {
         [TestMethod]
-        public void NotEnoughPathSegmentsTest()
+        public void NotEnoughPathSegments_ThrowsException_Test()
         {
-            try
-            {
-                var serviceBundle = TestCommon.CreateDefaultServiceBundle();
-                var instance = Authority.CreateAuthority("https://login.microsoftonline.in/tfp/");
-                Assert.IsNotNull(instance);
-                Assert.AreEqual(instance.AuthorityInfo.AuthorityType, AuthorityType.B2C);
-
-                Assert.Fail("test should have failed");
-            }
-            catch (Exception exc)
-            {
-                Assert.IsInstanceOfType(exc, typeof(ArgumentException));
-                Assert.AreEqual(MsalErrorMessage.B2cAuthorityUriInvalidPath, exc.Message);
-            }
+            var ex = Assert.ThrowsException<ArgumentException>(() =>
```

Comment:
Explicit type?

Reply:
We don't really have a strict rule of implicit vs explicit variable typing. Generally we use `var`s unless it's more readable to use an explicit type. Here it's clear that it's `ArgumentException`.